//CONDICIONAIS
try{
  console.log(result)
}catch(error){
  console.error(`Não foi possível acessar o resultado Error: ${error}`)
}finally{
  console.log('Procedimento finalizado')
}
